src/main/java/xRoss.java

```diff
@@ -48,15 +48,15 @@ public static void main(String[] args) {
         // boolean value on whether
         boolean continueLoop = true;
 
-        while (continueLoop){
+        while (continueLoop) {
```

Comment:
Since there are so many if conditions, I think you can consider using switch statement to implement your code. In the same time, I think you can also consider extract the whole codes inside this while loop to a new method, and name it like readUserInput or readCommand. So that you won't have long codes inside your main method :)